src/olympia/addons/models.py

```diff
@@ -187,7 +186,8 @@ def featured(self, app, lang=None, type=None):
         Filter for all featured add-ons for an application in all locales.
         """
         ids = get_featured_ids(app, lang, type)
-        return manual_order(self.listed(app), ids, 'addons.id')
+        qs = manual_order(self.listed(app), ids, 'addons.id')
+        return qs
```

Comment:
not that it matters, but this isn't a necessary change

Reply:
Yep, that was a leftover from some debugging I did, removed in 23dcd85a29a21c098ebdb6029fc9b20f86aeb2fb